examples/suit_update/README.md

```diff
@@ -465,22 +465,24 @@ displayed during this step:
 Once the new image is written, a final validation is performed and, in case of
 success, the application reboots on the new slot:
 
+    Finalizing payload store
     Verifying image digest
-    riotboot: verifying digest at 0x1fffbd15 (img at: 0x1000 size: 77448)
+    Starting digest verification against image
+    Install correct payload
     Verifying image digest
-    riotboot: verifying digest at 0x1fffbd15 (img at: 0x1000 size: 77448)
-    suit_parse() success
-    SUIT policy check OK.
-    suit_coap: finalizing image flash
-    riotboot_flashwrite: riotboot flashing completed successfully
+    Starting digest verification against image
+    Install correct payload
     Image magic_number: 0x544f4952
-    Image Version: 0x5e71f662
-    Image start address: 0x00001100
-    Header chksum: 0x745a0376
+    Image Version: 0x5fa52bcc
+    Image start address: 0x00201400
+    Header chksum: 0x53bb3d33
+    suit_coap: rebooting...
 
-    main(): This is RIOT! (Version: 2020.04)
+    main(): This is RIOT! (Version: 2020.10)
     RIOT SUIT update example application
-    running from slot 1
+    running from slot 0
```

Comment:
I think slot 1 was ok. What I meant is that "Running from slot 0" is now printed after the shell command `current-slot` is issued.
For the version I agree this should also be changed but maybe to something more generic `(Version: <version xx>)` or something the like.

Reply:
ping @fjmolinas

Reply:
@fjmolinas this still needs to be addressed IMHO.

Reply:
No, its still the same since there is the following in the startup

```
    cmd_print_current_slot(0, NULL);
    cmd_print_riotboot_hdr(0, NULL);
```

```
2020-11-16 11:49:45,398 # main(): This is RIOT! (Version: 2021.01-devel-621-g0b449-pr_suit_interactive_test)
2020-11-16 11:49:45,402 # RIOT SUIT update example application
2020-11-16 11:49:45,403 # Running from slot 0
2020-11-16 11:49:45,406 # Image magic_number: 0x544f4952
2020-11-16 11:49:45,408 # Image Version: 0x5fb25915
2020-11-16 11:49:45,411 # Image start address: 0x00002400
2020-11-16 11:49:45,413 # Header chksum: 0x28e77a69

```

But I see that the slot should indeed be 1.